R/bootstrap.nlsfit.R

```diff
@@ -306,15 +323,15 @@ bootstrap.nlsfit <- function(fn,
     if (missing(CovMatrix)) {
       InvCovMatrix <- try(invertCovMatrix(bsamples, boot.l = 1, boot.samples = TRUE, cov_fn = cov_fn), silent = TRUE)
       inversion.worked(InvCovMatrix)
-      dY <- chol(InvCovMatrix)
+      W <- chol(InvCovMatrix)
```

Comment:
I like this renaming! `dY` being `dy` and `1/dy` in different parts of the function was rather irritating.